device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C256S1/buffers_defaults_objects.j2

```diff
@@ -83,40 +86,93 @@
             "size":"9216",
             "dynamic_th":"-6"
         },
-        "queue0_lossy_profile": {
-            "pool":"egress_lossy_pool",
-            "size":"0",
-            "dynamic_th":"0"
+        "queue0_downlink_lossy_profile": {
+            "dynamic_th": "0",
+            "size": "0",
+            "pool": "egress_lossy_pool"
         },
-        "queue1_lossy_profile": {
-            "pool":"egress_lossy_pool",
-            "size":"0",
-            "dynamic_th":"0"
+        "queue1_downlink_lossy_profile": {
+            {% if switch_trimming_enable %}
+            "packet_discard_action": "trim",
+            {% endif %}
+            "pool": "egress_lossy_pool",
+            "size": "0",
+            "dynamic_th": "-7"
```

Comment:
let's use dynamic 0 as HBA is not disabled

Reply:
Done